lib/ansible/modules/cloud/amazon/aws_api_gateway.py

```diff
@@ -30,6 +30,11 @@
 version_added: '2.4'
 requirements: [ boto3 ]
 options:
+  name:
+    description:
+      - The name of the API to manage.  If api_id is None, then the name will be
```

Comment:
api_id should be formatted as a module parameter (i.e. `I(api_id)`)

From http://docs.ansible.com/ansible/latest/dev_guide/developing_modules_documenting.html#formatting-options